Hudl.Mjolnir/Config/BulkheadConfigurationObserverCreationExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Hudl.Mjolnir.Config
+{
+    internal static class BulkheadConfigurationObserverCreationExtensions
```

Comment:
Minor: Since these are extensions on `BulkheadConfiguration`, I'd expect this to be named `BulkheadConfigurationExtensions`.